### DISCUSSION TITLE
Is it possible to put policy into descriptor?

### DISCUSSION THREAD

**Discussion by dr-orlovsky** [rust-bitcoin/rust-miniscript#332] [Q&A]
Specifically when parsing string representation of the descriptor.

**Comment by sanket1729** [comment]
Can you elaborate with an example? Do you mean using the `policy` string as `Pk`?

**Reply by dr-orlovsky** [reply]
For instance, not write `tr(acc0,{multi_a(...),{and_v(v:multi_a(...),after(10)),and_(v:multi_a(...),after(100))}})` but instead `tr(acc0,{multi_a(...),{and(multi_a(...),after(10)),and(multi_a(...),after(100))}})`
